c_src/filewatch_inotify.c

```diff
@@ -149,9 +167,55 @@ static ErlDrvSSizeT call(
 
   fail_alloc:
   fail_decode:
+  fail_op_check:
     return -1;
 }
 
+/* Return type to Erlang:
+   {port(), [{watch_descriptor(), filename:basename()}]}
+*/
+static void send_output(ErlDrvData self_)
+{
+    struct instance *self = (struct instance *)self_;
+
+    size_t pos = 0;
+    ErlDrvTermData buf[(MAX_EVENTS * EVENT_TUPLE_LENGTH) + 3];
```

Comment:
It's a bit hard to tell what `+3` refers to. A comment or a named constant would be helpful.